components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -234,6 +234,61 @@ def create_db_tables(
     logger.info(f"Created {component_name} tables.")
 
 
+def create_results_cache_indexs(
+    instance_id: str,
+    clp_config: CLPConfig,
+    container_clp_config: CLPConfig,
+    mounts: CLPDockerMounts,
+):
+    component_name = RESULTS_CACHE_COMPONENT_NAME
+    logger.info(f"Creating {component_name} indexes...")
+
+    container_name = f"clp-{component_name}-indexes-creator-{instance_id}"
+
+    # Create results cache config file
+    results_cache_config_filename = f"{container_name}.yml"
+    results_cache_config_file_path = clp_config.logs_directory / results_cache_config_filename
+    with open(results_cache_config_file_path, "w") as f:
+        yaml.safe_dump(container_clp_config.results_cache.dict(), f)
```

Comment:
this is to be consistent with MySQL database flow. but it is true that there's less parameters for results cache. @kirkrodrigues any preference?

Reply:
Since we're not using auth for MongoDB, sure, makes sense to simplify things and get rid of the config file. For MySQL, it's different since we don't want to pass credentials through the CLI (we could change that to env-vars, but separate issue).